host/class/uvc/usb_host_uvc_2/uvc_host.c

```diff
@@ -0,0 +1,816 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <inttypes.h>
+#include <limits.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/queue.h>
+
+#include "esp_log.h"
+#include "esp_check.h"
+#include "esp_system.h"
+
+#include "usb/usb_host.h"
+#include "usb/uvc_host.h"
+#include "uvc_control.h"
+#include "uvc_stream.h"
+#include "uvc_types_priv.h"
```

Comment:
Why do we need `_priv.h` postfix in the filenames? 
They are already in the private folder. Or am I missing something? 

I mean, I just bumped in that when I tried to find the file, related to "uvc_frame.c" which is usually has the same name, but header. I didn't find it and then I realized that I need to search "_frame_priv.h" )
Just a bit confusing IMHO.

But maybe there is an idea behind such naming.